spec/unit/asset_sync_spec.rb

```diff
@@ -65,6 +65,17 @@
     it "should default log_silently to true" do
       expect(AssetSync.config.log_silently).to be_truthy
     end
+    
+    it "log_silently? should reflect the configuration" do
+      AssetSync.config.log_silently = false
+      expect(AssetSync.config.log_silently?).to eq(false)
+    end
+
+    it "log_silently? should always be true if ENV['RAILS_GROUPS'] == 'assets'" do
+      AssetSync.config.log_silently = false
+      expect(ENV).to receive(:[]).with('RAILS_GROUPS').and_return('assets')
```

Comment:
OK I think it's fine to have `expect`
But I think you can add comment explaining what that line does (stub + expect)
It's easier for people to read comments than finding history => PR => Line note :)